chia/data_layer/s3_plugin_service.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import annotations
+
+import asyncio
+import concurrent.futures
+import functools
+import logging
+import sys
+from pathlib import Path
+from typing import Any, Dict, List
+from urllib.parse import urlparse
+
+import boto3 as boto3
+import yaml
+from aiohttp import web
+from botocore.exceptions import ClientError
+
+from chia.types.blockchain_format.sized_bytes import bytes32
+
+log = logging.getLogger(__name__)
```

Comment:
As it is now, this looks to be unused.  Maybe we should code it to log to stdout by default instead of using `print()`?  This could be a separate future change.

Reply:
Logging to a file is one of the items on Ken's list, so I think we leave this as is and make some followup PRs to use it